itsnameissteven/Refactor-Tractor-Fitlit
# Feature/activity class

## Pull Request

### Description
* Is this a feature or a fix?
This is a fix
* Where should the reviewer start?
Review the activity test and activity.js file
* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
Activity Test was refactored utilizing less data.  Added additional tests 
* What outside features or research did you use to implement this fix?
n/a
* How should this be tested?
Run npm test to make sure all tests still pass. Open web app to ensure everything still loads properly
### Relevant Screen Shots
n/a
***
#### Please review considerations below:
